project_team_leave_management/views/res_partner_view.xml

```diff
@@ -64,7 +64,7 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">res.partner</field>
         <field name="view_mode">kanban,tree,form</field>
-        <field name="domain">[('parent_id', '=', False)]</field>
+        <field name="domain">[('parent_id', '=', False),('supplier_rank','>=',1)]</field>
```

Comment:
The domain can be kept as [('parent_id', '=', False),('supplier_rank','>',0)]
For supplier_rank , it can checked as > 0 rather than >=1